templates/tom_targets/partials/galaxy_table.html

```diff
@@ -0,0 +1,39 @@
+{% load tom_common_extras targets_extras target_list_extras %}
+
+<table class="table table-hover">
+  <thead>
+    <tr>
+      <th scope="col">PCC</th>
+      <th scope="col">R.A.</th>
+      <th scope="col">Dec.</th>
+      <th scope="col">Dist.</th>
+      <th scope="col">Err.</th>
+      <th scope="col">Mag</th>
+      <th scope="col">Filter</th>
+      <th scope="col">Source</th>
+    </tr>
+    <tr>
+      <th> </th>
+      <td>deg.</td>
+      <td>deg.</td>
+      <td>Mpc</td>
+      <td>Mpc</td>
+      <td>AB</td>
+      <td> </td>
+      <td> </td>
+    </tr>
+  </thead>
+  <!-- <tr>{{ target|target_extra_field:"Host Galaxies"}}</tr> -->
+  {% for galaxy in target|get_host_galaxies %}
```

Comment:
It would be nice to add a case for when there are no host galaxies, so that it's not just blank.